services/ui-src/src/components/pages/Export/McparPdfExport.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box, Heading } from "@chakra-ui/react";
+import { StickyBanner, ReportContext, Table, FieldsSection } from "components";
+
+import { useContext } from "react";
+import { convertDateUtcToEt } from "utils";
+
+import addEditProgram from "../../../forms/addEditProgram/addEditProgram.json";
+
+export const McparPdfExport = () => {
+  const data = useContext(ReportContext);
```

Comment:
if we only ever use `report` could we destructure this to go straight to report?